oaknational/Oak-Web-Application
# Define curriculum pages

## Description

- Create internal pages
- Update breadcrumbs and tab links on curriculum pages
- Update style for current tab

## How to test

1. Go to https://deploy-preview-1824--oak-web-application.netlify.thenational.academy/beta/teachers/curriculum/secondary-geography/overview
2. Click on tab `Unit sequence`
3. You should see the tab content update and the new tab highlighted

## Checklist
